build_image

```diff
@@ -56,12 +56,13 @@ FLAGS_HELP="USAGE: build_image [flags] [list of images to build].
 This script is used to build a CoreOS image. CoreOS comes in many
 different forms.  This scripts can be used to build the following:
 
-prod - Production image for CoreOS. This image is for booting.
+prod - Production image for CoreOS. This image is for booting (default if no argument is given).
+prodtar - Production container tar ball (implies prod). Usable by machinectl import-tar or with docker import.
```

Comment:
+ "This can e.g. be used to run the Flatcar production image in a container."

Reply:
Updated it, but changed it to *as a container* because the image is run as a container and not inside one.

Reply:
Good catch, thank you :smile: